test/lib/asserter/README.md

```
@@ -2,21 +2,167 @@
+string a("hello assert");
+assert(a.empty());  // Just tells you the assertion failed
 ```
+
+**With asserter:**
+```cpp
 string a("hello assert");
+ASSERT(a.empty())(a)(a.size());  // Shows you WHY it failed
+```
+
+Output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)